File: FCUBEAPP/src/app/models/menureportaccessmodel.ts
export class Menureportaccessrightsmodel {
accountID: string = "";
  accountName: string = "";
  approveYn: string = "";
  level: string = "";
  parentAccountID?: string | null = null;
  children: Menureportaccessrightsmodel[] = [];
  checked: boolean = false;
  isExpanded: boolean = false;
}

File: OMKARAUTOAPI/FinTrans/Models/MenuReportAccessModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FinTrans.Models
{
    public class MenuReportAccessModel
    {
        public string? AccountID { get; set; }
        public string? AccountName { get; set; }
        public string? ApproveYn { get; set; }
        public string? Level { get; set; }
        public string? ParentAccountID { get; set; }
        public List<MenuReportAccessModel> Children { get; set; } = new List<MenuReportAccessModel>();
    }
}
